docs/user-guide/examples/advanced_examples/advanced_orchestration.md

```diff
@@ -7,4 +12,342 @@
+
+$$ M \sim U(a,b) $$
+
+Where $M$ is the measured voltage, $a$ is the lower bound for possible values, and $b$ is the upper bound for possible values. This is read as, "$M$
+is distributed uniformly with bounds $a$ and $b$."
+
+![](../../../img/uniform_dist.png)
+
+The uniform distribution is among the most simple of probability distributions. Additional resources on probability and statistics are plentiful; [Statistical Rethinking](https://xcelab.net/rm/statistical-rethinking/) is highly recommended.
+
+### Monte Carlo Co-sim Example: EV Garage Charging
+
+The example co-simulation to demonstrate Monte Carlo distribution sampling is that of an electric vehicle (EV) charging garage. Imagine a parking garage that only serves EVs, has a static number of charging ports, and always has an EV connected to a charging port. An electrical engineer planning to upgrade the distribution transformer prior to building the garage may ask the question: What is the likely power draw that EVs will demand?
+
+#### Probability Distributions
+
+_Likely_ is synonymous for _probability_. As we are interested in a probability, we cannot rely on a deterministic framework for modeling the power draw from EVs. I.e., we cannot assume that we know a priori the exact demand for Level 1, Level 2, and Level 3 chargers in the garage. A deterministic assumption would be equivalent to stating, e.g., that 30% of customers will need Level 1 charge ports, 50% will need Level 2, and 20% will need Level 3. What if, instead of static proportions, we assign a distribution to the need for each level of charge port. The number of each level port is discrete (there can't be 0.23 charge ports), and we want the number to be positive (no negative charge ports), so we will use the [Poisson distribution](https://en.wikipedia.org/wiki/Poisson_distribution). The Poisson distribution is a function of the anticipated average of the value $\lambda$ and the number of samples $k$. Then we can write the distribution for the number of chargers in each level, $L$, as
+
+$$ L \sim P(k,\lambda) $$
+
+Let's extend our original assumption that the distribution of chargers is static to Poisson distributed, and let's assume that there are 100 total charging ports:
+
+$$ L1 \sim P(100,0.3) $$
+$$ L2 \sim P(100,0.5) $$
+$$ L3 \sim P(100,0.2) $$
+
+![](../../../img/EVPoisson.png)
+
+What if we weren't entirely certain that the average values for $L1, L2, L3$ are $0.3, 0.5, 0.2$, we can also sample the averages from a normal distribution centered on these values with reasonable standard deviations. We can say that:
+
+$$ \lambda \sim N(\mu,\sigma) $$
+
+Which means that the input to $L$ is distributed normally with average $\mu$ and standard deviation $\sigma$.
+
+Our final distribution for modeling the anticipated need for each level of charging port in our $k = 100$ EV garage can be written as:
+
+$$ L \sim P(k,\lambda) $$
```

Comment:
I think we need to add a line to the `docs/conf.py` file to configure the MyST-parser to work with dollarmath: https://myst-parser.readthedocs.io/en/latest/syntax/optional.html#math-shortcuts

Reply:
OK, if you say so. I've never worked with the build process for the docs so somebody else (@phlptp?, @nightlark?) so I propose somebody who knows what they are doing tackle this.

Reply:
Seems to work now. MyST extensions for math are enabled, and an old conflicting Sphinx markdown table extension has been removed.